spyder/plugins/workingdirectory/plugin.py

```diff
@@ -54,7 +53,7 @@ class WorkingDirectory(SpyderPluginV2):
 
     # --- Signals
     # ------------------------------------------------------------------------
-    sig_current_directory_changed = Signal(str, str)
+    sig_current_directory_changed = Signal(str, str, str)
```

Comment:
- Please document the new parameter in the docstring.
- Also, expand the entry about this signal's change in our Changelog.